lighthouse-core/audits/byte-efficiency/render-blocking-resources.js

```diff
@@ -91,6 +103,7 @@ class RenderBlockingResources extends Audit {
       node.traverse(node => deferredNodeIds.add(node.id));
 
       // "wastedMs" is the download time of the network request, responseReceived - requestSent
+      // @ts-ignore - TODO(phulce): nodeTiming.startTime/endTime shouldn't be optional by this point?
```

Comment:
yeah we'll probably need to do an `IntermediateNodeTiming` vs. `NodeTiming` split, I can do that in followup 👍